mdm/common/src/main/java/com/gentics/mesh/core/data/dao/PersistingNodeDao.java

```diff
@@ -1283,9 +1283,6 @@ default boolean update(HibProject project, HibNode node, InternalActionContext a
 		// Check whether this is the first time that an update for the given language and branch occurs. In this case a new container must be created.
 		// This means that no conflict check can be performed. Conflict checks only occur for updates on existing contents.
 		if (latestDraftVersion == null) {
-			// Create a new field container
```

Comment:
May introduce an inconsistency in ODB when parent node check failed, but the container is already created, belonging to no one.